gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -0,0 +1,468 @@
+      futures.add(
+          executor.submit(
+              () -> {
+                FileStatus fileStatus = fs.getFileStatus(testFile);
+                long fileSize = fileStatus.getLen();
+                long maxReadPositionExclusive = fileSize - readSize + 1;
+
+                LongSummaryStatistics openLatencyNs = newLongSummaryStatistics(openLatencyNsList);
+                LongSummaryStatistics seekLatencyNs = newLongSummaryStatistics(seekLatencyNsList);
+                LongSummaryStatistics readLatencyNs = newLongSummaryStatistics(readLatencyNsList);
+                LongSummaryStatistics closeLatencyNs = newLongSummaryStatistics(closeLatencyNsList);
+
+                ThreadLocalRandom random = ThreadLocalRandom.current();
+                byte[] readBuffer = new byte[readSize];
+
+                initLatch.countDown();
+                startLatch.await();
+                try {
+                  for (int j = 0; j < numOpen; j++) {
+                    try {
+                      long seekPos = random.nextLong(maxReadPositionExclusive);
+
+                      long openStart = System.nanoTime();
+                      FSDataInputStream input = fs.open(testFile);
+                      openLatencyNs.accept(System.nanoTime() - openStart);
+                      try {
+                        for (int k = 0; k < numReads; k++) {
+                          long seekStart = System.nanoTime();
+                          input.seek(seekPos);
+                          seekLatencyNs.accept(System.nanoTime() - seekStart);
+
+                          long readStart = System.nanoTime();
+                          int numRead = input.read(readBuffer);
+                          readLatencyNs.accept(System.nanoTime() - readStart);
+
+                          if (numRead != readSize) {
+                            System.err.printf(
+                                "Read %d bytes from %d bytes at offset %d!%n",
+                                numRead, readSize, seekPos);
+                          }
+                        }
+                      } finally {
+                        long closeStart = System.nanoTime();
+                        input.close();
+                        closeLatencyNs.accept(System.nanoTime() - closeStart);
+                      }
+                    } catch (Throwable e) {
+                      logger.atSevere().withCause(e).log("Failed random read from '%s'", testFile);
+                    }
+                  }
+                } finally {
+                  stopLatch.countDown();
+                }
+
+                return null;
+              }));
+    }
+    executor.shutdown();
+
+    awaitUninterruptibly(initLatch);
+    long startTime = System.nanoTime();
+    startLatch.countDown();
+    awaitUninterruptibly(stopLatch);
+    double runtimeSeconds = nanosToSeconds(System.nanoTime() - startTime);
+    long operations = combineStats(readLatencyNsList).getCount();
```

Comment:
How is combining the latencies converted to number of operations?

Reply:
Each latency in the list has `count` so it will be just a sum of all individual threads counts.